docs/pages/configuration/hooks/basics.mdx

```diff
@@ -235,6 +235,23 @@ hooks:
   events: ["after:build"]
 ```
 
+## Execute hooks once
+
+Hooks can be executed once per deployment when executing a command in a container. This can be useful for running one-time development tasks without init containers. In the following example, the command would only run once in containers matching the `nginx` image selector.
```

Comment:
"in containers matching the `nginx` image selector" => "in all containers running with the image `nginx`"

Ideally, we should add a tag or use a different image than `nginx` to make it easier to understand because this implies tag `latest` which is not a best practice

Reply:
Changed to `nginx:1.21` to avoid implying `latest`